.github/workflows/ci.yml

```
@@ -6,10 +6,10 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

I think we probably want to change this back to `ubuntu-latest`.

```suggestion
    runs-on: ubuntu-latest
```

---

I kept `macos-latest`. 
https://github.com/momentohq/client-sdk-javascript/pull/46#issuecomment-1078653291